packages/docs/src/helpers/githubUrl.js

```diff
@@ -0,0 +1,10 @@
+import join from 'url-join';
+
+const cmsdsGithubUrl = 'https://github.com/CMSgov/design-system';
+
+function githubUrl(pathname = '') {
+  const ghUrl = process.env.githubUrl || cmsdsGithubUrl;
```

Comment:
[suggestion] Consider removing this `ghUrl` variable. It seems like this might have been used originally when the child DS's were separate repos (see [this example](https://github.com/CMSgov/hcgov-design-system/blob/master/cmsds.config.js#L12)). Now that we're in a monorepo, this url should always be `'https://github.com/CMSgov/design-system'`